mixed-content/generic/common.js

```diff
@@ -0,0 +1,238 @@
+function normalizePort(targetPort) {
```

Comment:
Sorry that I didn't ask you to add comments to the `referrer-policy` code. I didn't do a great job reviewing... I'll pay more attention this time. :)

Please add comments to this code. I'd suggest following the guidelines at http://google.github.io/styleguide/javascriptguide.xml#Comments.